client/engine/store/persiststore.go

```diff
@@ -468,3 +471,36 @@ func (ps *PersistStore) checkError(err error) {
 		panic(err)
 	}
 }
+
+func (ps *PersistStore) SetVoucherInfo(channelId types.Destination, vs payments.VoucherInfo) error {
```

Comment:
nit: why `vs` instead of `v` or just `voucher`? (I think the greatly-abbreviated convention is more generally for receiver variables than inputs).

Reply:
Good catch. Originally I had named `VoucherInfo` `VoucherStatus` which is where the abbreviation `vs` comes from. I'll probably rename to `v` or `vouchInfo`

Reply:
> I think the greatly-abbreviated convention is more generally for receiver variables than inputs

Technically I think the recommended convention is that short variable names should be preferred everywhere, including [parameters](https://go.dev/talks/2014/names.slide#7). Personally I'm in favor of longer, descriptive names 🤷

Reply:
I've renamed to `v` in https://github.com/statechannels/go-nitro/pull/1164/commits/1f9c7cbcf8e36e8cd207122b3ba6cd654c0424c2